StRoot/StETofUtil/etofHelperFunctions.cxx

```diff
@@ -0,0 +1,305 @@
+#include <cmath>
+
+#include "StETofUtil/etofHelperFunctions.h"
+
+#include "SystemOfUnits.h"
+#include "PhysicalConstants.h"
+
+#include "StMessMgr.h"
+#include "StBTofHeader.h"
+
+
+// *******************************************************************************************************
+//
+double
+calculateVpdTstart( const StBTofHeader* btofHeader, const double& pVtxZ, const double& bTofClockRange )
+{
+    if( !btofHeader ) {
+        LOG_WARN << "calculateVpdTstart():  -- no bTof header. Skip start time calculation." << endm;
+        return 0.;
+    }
+
+
+    const int nVpd = 19; // number of VPD tubes on each side of STAR
```

Comment:
Isn't this constant already defined somewhere else?

Reply:
I guess but I couldn't figure out where.